### PR TITLE
Updating README and creating gitignore

### PR DESCRIPTION
**Pull request by LuanComputacao**
A lot of modifications

- README
- Favicons
- Images
- Google Analytics
- JS bug fixing
- HTML fixing
- CSS fixing

### CODE REVIEW

**Comment by LuanComputacao** [comment]
They'll not work properly in both URLs

File: README.md [hunk 1]
```diff
@@ -67,6 +113,25 @@ There are 3 places to change, look at {UPPERCASE} sentences on the template:
...
+    <link rel="apple-touch-icon" sizes="144x144" href="../assets/icons/apple-icon-144x144.png">
+    <link rel="apple-touch-icon" sizes="152x152" href="../assets/icons/apple-icon-152x152.png">
+    <link rel="apple-touch-icon" sizes="180x180" href="../assets/icons/apple-icon-180x180.png">
+    <link rel="icon" type="image/png" sizes="192x192" href="../assets/icons/android-icon-192x192.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="../assets/icons/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="96x96" href="../assets/icons/favicon-96x96.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="../assets/icons/favicon-16x16.png">
+    <link rel="manifest" href="../assets/icons/manifest.json">
+    <meta name="msapplication-TileColor" content="#ffffff">
+    <meta name="msapplication-TileImage" content="assets/icons/ms-icon-144x144.png">
```
Comment (ramiroluz) [inline README.md]: Faltou esse usar caminho relativo?

File: README.md [hunk 2]
```diff
@@ -114,7 +179,7 @@ There are 3 places to change, look at {UPPERCASE} sentences on the template:
 
       <div class="row">
         <div class="col-xs-12">
-          <div markdown-it ng-include="'../docs/pt/{NAME AND PATH OF THE FILE-EXAMPLE->contact.pt.md}'" class="md">
+          <div markdown-it ng-include="'/docs/pt/{NAME AND PATH OF THE FILE-EXAMPLE->contact.pt.md}'" class="md">
```
[the remark this answers is not in the document]
Comment (ramiroluz) [reply README.md]: :point_up:

File: assets/css/custom.css [hunk 3]
```diff
@@ -931,8 +931,7 @@ h1, h2, h3, h4 {
   animation-duration: 10s;
   animation-iteration-count: infinite;
   animation-timing-function: linear;
-  background-image: url('../img/satellite.png');
-  background-repeat: no-repeat;
+  background: url('/assets/img/satellite.svg') no-repeat;
```
[the remark this answers is not in the document]
Comment (ramiroluz) [reply assets/css/custom.css]: :point_up_2:

File: assets/views/head.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
```
Comment (ramiroluz) [inline assets/views/head.html]: This file isn't using relative paths.

File: faq/index.html [hunk 5]
```diff
@@ -16,6 +17,25 @@
...
+    <link rel="apple-touch-icon" sizes="144x144" href="../assets/icons/apple-icon-144x144.png">
+    <link rel="apple-touch-icon" sizes="152x152" href="../assets/icons/apple-icon-152x152.png">
+    <link rel="apple-touch-icon" sizes="180x180" href="../assets/icons/apple-icon-180x180.png">
+    <link rel="icon" type="image/png" sizes="192x192" href="../assets/icons/android-icon-192x192.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="../assets/icons/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="96x96" href="../assets/icons/favicon-96x96.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="../assets/icons/favicon-16x16.png">
+    <link rel="manifest" href="../assets/icons/manifest.json">
+    <meta name="msapplication-TileColor" content="#ffffff">
+    <meta name="msapplication-TileImage" content="assets/icons/ms-icon-144x144.png">
```
Comment (ramiroluz) [inline faq/index.html]: :point_up:

File: faq/index.html [hunk 6]
```diff
@@ -25,86 +45,95 @@
...
+</nav>
 
-      <div class="row">
+<div class="container" ng-app="scipyla.md" id="container">
+
+    <div class="row">
         <div class="col-xs-12">
-          <div markdown-it ng-include="'../docs/pt/faq.pt.md'" class="md">
-          </div>
+            <div markdown-it ng-include="'/docs/pt/faq.pt.md'" class="md">
```
Comment (ramiroluz) [inline faq/index.html]: I believe it was working with relative path.

File: help/index.html [hunk 7]
```diff
@@ -16,6 +17,25 @@
...
+    <link rel="apple-touch-icon" sizes="144x144" href="../assets/icons/apple-icon-144x144.png">
+    <link rel="apple-touch-icon" sizes="152x152" href="../assets/icons/apple-icon-152x152.png">
+    <link rel="apple-touch-icon" sizes="180x180" href="../assets/icons/apple-icon-180x180.png">
+    <link rel="icon" type="image/png" sizes="192x192" href="../assets/icons/android-icon-192x192.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="../assets/icons/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="96x96" href="../assets/icons/favicon-96x96.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="../assets/icons/favicon-16x16.png">
+    <link rel="manifest" href="../assets/icons/manifest.json">
+    <meta name="msapplication-TileColor" content="#ffffff">
+    <meta name="msapplication-TileImage" content="assets/icons/ms-icon-144x144.png">
```
Comment (ramiroluz) [inline help/index.html]: :point_up:

File: help/index.html [hunk 8]
```diff
@@ -25,86 +45,96 @@
...
+</nav>
 
-      <div class="row">
+<div class="container" ng-app="scipyla.md" id="container">
+
+    <div class="row">
         <div class="col-xs-12">
-          <div markdown-it ng-include="'../docs/pt/help.pt.md'" class="md">
-          </div>
+            <div markdown-it ng-include="'/docs/pt/help.pt.md'" class="md">
```
Comment (ramiroluz) [inline help/index.html]: :point_up:

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (LuanComputacao) [reply README.md]: Removed from the README file

File: assets/css/custom.css (hunk 3 above)
[the remark this answers is not in the document]
Comment (LuanComputacao) [reply assets/css/custom.css]: Fixed

File: README.md (hunk 1 above)
Comment (LuanComputacao) [reply README.md]: Removed from the README file

File: assets/views/head.html (hunk 4 above)
Comment (LuanComputacao) [reply assets/views/head.html]: My mistake.
That was a test that I'll delete

File: faq/index.html (hunk 5 above)
Comment (LuanComputacao) [reply faq/index.html]: fixed

File: faq/index.html (hunk 6 above)
Comment (LuanComputacao) [reply faq/index.html]: yes

File: help/index.html (hunk 7 above)
Comment (LuanComputacao) [reply help/index.html]: fixed

File: help/index.html (hunk 8 above)
Comment (LuanComputacao) [reply help/index.html]: fixed

**Pull request merged**